css/index.css

```diff
@@ -1,3 +1,293 @@
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.container{
+	position: relative;
+	width:990px;
+	padding: 50px 62px 50px 62px;
+	border: 1px;
+	margin: 0 auto;
+}
+nav{
+	display: flex;
+	justify-content: space-between;
+	padding-bottom: 20px;
+	color: gray;
+}	
+nav a{
+	color: gray;
+  text-decoration: none;
+}
+nav a:link{
+	color: gray;
+}	
+nav a:visited{
+	color: gray;
+}	
+nav a:hover{
+	color: lightgray;
+}	
+ul{
+	padding-top: 10px;
+}
+li{
+	display: inline;
+	padding-right: 46px;
+	vertical-align: middle;
+}
+.banner{
+	display: flex;
+	position: relative;
+	width: 100%;
+	padding-top: 40px;
+    margin-bottom: 60px;
+
+}
+.banner h1{
+	position: relative;
+	font-size: 72px;
+	text-align: center;
+
+}
+.banner button{
+	display: block;
+	position: relative;
+	padding: 5px 30px;
+	background-color: white;
+	border: 1px solid;
+	margin: auto;
+}
+section:first-of-type{
+	border-top: 1px solid;
+}
+section{
+	margin: 0px; 
+	padding: 30px 0;
+}
+main img:last-of-type{
+	position: relative;
+	display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+section:last-of-type{
+	border-bottom: 1px solid;
+}
+.top-content{
+	display: inline-block;
+	width: 49%;
+	margin-right: 6px;
+}
+.bottom-content{
+	display: inline-block;
+	width: 32%;
+	margin-right: 9px;
+}
+.header{
+	width: 49%;
+	margin: 0 auto;
+}
+footer{
+	padding: 30px 0;
+}
+footer .copyright{
+	display: block;
+	text-align: center;
+	padding-top: 50px;
+}
+/* tablet version*/ 
+@media(max-width: 800px){
+    .container{
+        width: 100%;
+    }         
+    
+    nav{
+      display: flex;
+      flex-direction: column-reverse;
+      justify-content: space-between;
+      width: 100%;
+      padding-bottom: 5%;
+      color: gray;
+    }
+
+    .logo{
+        width: 40%;
+        align-self: center;
+    }
+    li{
+        padding: 1%;
+    }
+
+    ul{
+      display: flex;
+      justify-content: space-evenly;
+    }
+    
+    .banner img{
+        display: none;
+    }
+
+    .middle-img{
+        width: 100%;
+    }
+
+    .banner{
+
+        flex-direction: column;
+    }
+    
+    .top-content{
+        display: flex;
+        flex-direction:column;
+        width: 100%;
+        justify-content: space-evenly;
+    }            
+
+    section {
+        display: flex;
+        align-content: space-between;
+    }
+}
+
+/* mobile version*/
+    @media(max-width: 500px){
```

Comment:
![Screen Shot 2019-04-02 at 1 49 59 PM](https://user-images.githubusercontent.com/8962594/55435199-39615180-554e-11e9-922f-e0c22da8c071.png)


It seems like you had inherited a padding left and right. You can overwrite them when you get to mobile view with `padding-left: 0` and `padding-right: 0` or `padding: 0`